ssz-rs/examples/serializable.rs

```diff
@@ -0,0 +1,80 @@
+use ssz_rs::prelude::*;
+
+#[derive(PartialEq, Eq, Debug, Default, Serializable)]
+struct SingleFieldTestStruct {
+    a: u8,
+}
+
+#[derive(PartialEq, Eq, Debug, Default, Serializable)]
+struct SmallTestStruct {
+    a: u16,
+    b: u16,
+}
+
+#[derive(PartialEq, Eq, Debug, Default, Clone, Serializable)]
+struct FixedTestStruct {
+    a: u8,
+    b: u64,
+    c: u32,
+}
+
+#[derive(PartialEq, Eq, Debug, Default, Clone, Serializable)]
+struct VarTestStruct {
+    a: u16,
+    b: List<u16, 1024>,
+    c: u8,
+}
+
+#[derive(PartialEq, Eq, Debug, Default, Serializable)]
+struct ComplexTestStruct {
+    a: u16,
+    b: List<u16, 128>,
+    c: u8,
+    d: List<u8, 256>,
+    e: VarTestStruct,
+    f: Vector<FixedTestStruct, 4>,
+    g: Vector<VarTestStruct, 2>,
+}
```

Comment:
instead of duplicating all of this code, let's just add one example struct to `examples/another_container.rs` that just derives `Serializable`

even just something like

```rust
#[derive(PartialEq, Eq, Debug, Default, Serializable)]
struct ComplexTestStruct {
    a: u16,
    b: List<u16, 128>,
    c: u8,
    d: List<u8, 256>,
    e: VarTestStruct,
    f: Vector<FixedTestStruct, 4>,
    g: Vector<VarTestStruct, 2>,
    h: u32,
}
```

and then show ser/de in that example code

Reply:
can you also add this example as a test to `ssz_rs::container::tests` and demonstrate a roundtrip?

that will give us some assurance the macro is working correctly